package.json

```diff
@@ -34,6 +34,7 @@
     "url": "http://github.com/azure/azure-rest-api-specs/issues"
   },
   "scripts": {
+    "postinstall": "scripts/switch-to-preproduction.sh",
```

Comment:
in this case, will log showed as orignal ci task pipelines? how to easily check task failure or not?

Reply:
The log will show "Switch to preproduction package.json" if PREPRODUCTION_PIPELINE is set. See the script content.